src/actions/socialLogin/FacebookActions.js

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+
+import {
+  FACEBOOK_AUTHENTICATION,
+  FACEBOOK_AUTH_SUCCESS,
+  FACEBOOK_AUTH_FAIL,
+} from '../socialLoginTypes';
+
+/**
+ * This is a function which returns the facebookSuccess action
+ * @param {object} - A object parameter type*
+ * @return {none}
+ * @example
+ * foo = () => {}
+ */
+const facebookSuccess = resp => ({
+  type: FACEBOOK_AUTH_SUCCESS,
+  payload: resp.data.user,
+});
+
+/**
+ * This is a function which returns the facebookFail action
+ * @param {object} - A object parameter type*
+ * @return {none}
+ * @example
+ * foo = () => {}
+ */
+const facebookFail = error => ({
+  type: FACEBOOK_AUTH_FAIL,
+  payload: error.response.data.errors,
+});
+
+/**
+ * This is a function which sends facebook registration token to the api
+ * @param {string} - A string param
+ * @return {none}
+ * @example
+ * foo = (string) => {}
+ */
+const FacebookLogin = token => (dispatch) => {
+  dispatch({
+    type: FACEBOOK_AUTHENTICATION,
+  });
+  // @type {object}
+  const config = {
+    headers: { 'Content-Type': 'application/json' },
+    responseType: 'application/json',
+  };
+  // @type {object}
+  const userData = {
+    access_token: token,
+  };
+  return axios
+    .post(
+      'https://ah-backend-kronos-staging.herokuapp.com/api/social-auth/facebook/',
+      userData,
+      config,
+    )
+    .then((resp) => {
+      sessionStorage.setItem('token', resp.data.user.token);
+      sessionStorage.setItem('user_email', resp.data.user.email);
+
+      dispatch(facebookSuccess(resp));
+      document.location.href = '/successpage';
```

Comment:
Navigate using ReactRouter

Reply:
@Yiga-fred this is a test page that is not essential to our application, I am using it to show that someone has been successfully logged in with that `email` linked to the social media account. So this does not require `react routing` as it is going to be removed after testing that the feature is functional.